### PR TITLE
Add proxy-permissions.xml file with documentation

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by fvanderbiest** [comment]
I've updated the documentation, hopefully without breaking the logic.
Please review again.

File: security-proxy/proxy-permissions.xml [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
-     Example with all tags:
+<!-- This file handles security settings for the internal proxy that we use
+     to workaround browser's same-origin policy.
+     The proxy can be reached at the /proxy/?url= endpoint.
+     These settings are important to prevent exploration of internal services
+     from the outside world !
 
+     Here are the settings:
+
+     allowByDefault: if set to true, the 'allowed' list of hosts will be checked
```
Comment (Vampouille) [inline security-proxy/proxy-permissions.xml]: "list of rules" instead of "list of hosts" ?

**Comment by fvanderbiest** [comment]
Will require an update to mention the IP range

**Comment by Vampouille** [comment]
> Will require an update to mention the IP range

Done with 4bca858

**Pull request merged**